src/Message/Toast.elm

```diff
@@ -41,21 +42,42 @@ label =
 -}
 decoder : Decoder Toast
 decoder =
+    Decode.oneOf
+        [ v2decoder
+        , v1Decoder
+        ]
+
+
+v2decoder : Decoder Toast
+v2decoder =
     (Decode.succeed Toast
-        |> optional "title" (Decode.maybe Decode.string) Nothing
+        |> required "title" Decode.string
         |> required "message" Decode.string
         |> optional "custom" (Decode.maybe Decode.value) Nothing
+        |> required "icon" Decode.string
     )
         |> expectLabel label
 
 
+v1Decoder : Decoder Toast
+v1Decoder =
+    (Decode.succeed Toast
+        |> optional "title" Decode.string "Notification!"
+        |> required "message" Decode.string
+        |> optional "custom" (Decode.maybe Decode.value) Nothing
+        |> hardcoded "info"
+    )
+        |> expectLabel "toastRequest"
```

Comment:
The v1 decoder provides default & fixed values for fields that were previously optional or not present.